lib/bolt/plugin/module.rb

```diff
@@ -12,15 +12,14 @@ def initialize(msg, plugin)
         end
       end
 
-      def self.load(name, modules, opts)
-        mod = modules[name]
+      def self.load(mod, opts)
         if mod&.plugin?
```

Comment:
The `&` is unnecessary here. We've guarded against `nil` at the single call site and can therefore incorporate that 
as a precondition of this method. And as the code stands now, if this argument _were_ ever `nil`, the `raise` would itself fail, because it calls `mod.name` directly.

Reply:
Updated.